src/pages/settings/sub/discord/discord_settings.html

```diff
@@ -101,6 +101,24 @@
                                             </div>
                                         </td>
                                     </tr>
+                                    <tr>
+                                        <td>
+                                            <span
+                                                i18n="i18n_LABEL_SETTINGS_DISCORD_USE_DEFAULT"
+                                            ></span>
+                                        </td>
+                                        <td class="right">
+                                            <div class="switch">
+                                                <label>
+                                                    <input
+                                                        id="settings-discord-use-default"
+                                                        type="checkbox"
+                                                    />
+                                                    <span class="lever"></span>
+                                                </label>
+                                            </div>
+                                        </td>
+                                    </tr>
```

Comment:
I feel that the wording for the Setting should rather be "Use alternative design" instead of use default and flip the switch on how the Setting functions.
People should automatically be opted into the Default value and be enabling the setting knowing what they're changing the format of the Rich Presences.

Additional might be worth doing a similar thing to what was implemented for the alternative design for Mini-player and using a small tooltip at the end of setting label to describe a bit of what is happening regarding the Alternative Layout.